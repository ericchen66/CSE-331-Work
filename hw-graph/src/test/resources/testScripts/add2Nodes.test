# Tests adding two nodes to a graph

# Creates an empty graph
CreateGraph graph1

# Adds two nodes
AddNode graph1 node2
AddNode graph1 node1

# Print the nodes in the graph
ListNodes graph1