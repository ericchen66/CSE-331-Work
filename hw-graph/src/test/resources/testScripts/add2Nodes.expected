# Tests adding two nodes to a graph

# Creates an empty graph
created graph graph1

# Adds two nodes
added node node2 to graph1
added node node1 to graph1

# Print the nodes in the graph
graph1 contains: node1 node2